assessments/0005-fluxcd.md

```diff
@@ -0,0 +1,183 @@
+
+Scale:
+- 1 = (Is not present or requires significant work)
+- 3 = (Is present, but needs work)
+- 5 = (Is executed extremely well or no improvement required)
+
+### Comments
+
+**Information architecture**: 
+
+- Flux's documentation is well organized, but there are improvements we could make:
+    - Between the [Guides](https://fluxcd.io/docs/guides/) and [Use cases](https://fluxcd.io/docs/use-cases/) overlap a bit. The topics in these two sections are around installing and using Flux with a variety of different technologies. The names of both are somewhat vague which causes a lack of clarity for the reader: they need to click in to understand what each section is about, and even after doing so it isn't entirely clear.
+    - The [Migration](https://fluxcd.io/docs/migration/) section isn't more important than [Toolkit components](https://fluxcd.io/docs/components/), [Toolkit dev guides](https://fluxcd.io/docs/gitops-toolkit/), or the [Flux CLI reference](https://fluxcd.io/docs/cmd/). Placing it above these in the information architecture doesn't make sense. 
+
+**New user content**: 
+
+- Flux has a thorough [Getting Started](https://fluxcd.io/docs/get-started/) page, complimented by a multi-CLI [Installation guide](https://fluxcd.io/docs/installation/) and an informative [Docs root](https://fluxcd.io/docs/concepts/). Together these three pages work well to introduce new users to Flux. No real improvements needed!
+
+**Content maintainability & site mechanics**: 
+
+- Documentation lives in the `flux/website` repository and uses Hugo+Docsy, which is our recommended stack! Full marks in this regard. There are two areas of potential concern:
+    The [Toolkit Components](https://fluxcd.io/docs/components/) section:
+        - The [documentation](https://github.com/fluxcd/website/tree/main/external-sources) is a touch opaque
+        - The [resulting documentation](https://fluxcd.io/docs/components/helm/helmreleases/) is not user friendly: it is difficult to read and difficult to understand. A useful statistic to keep in mind is that while 50% of developers learn best by looking at code and "hacking", the other 50% of developers want to understand high-level concepts better before beginning code: providing a bit more explanation up front, particularly around the object's required fields and what the expected values are. An example would be helpful for this audience. 
+    - Items under the **Project** top level navigation section. At a glance I couldn't figure out how these were being pulled in via (I assume) the [Community repository](https://github.com/fluxcd/community), meaning they suffer either from a lack of documentation or unclear automation.
+
+- Documentation versioning: Flux's documentation is not versioned but the tool itself has been versioned. 
+    - This leaves existing users of v1 who haven't upgraded in a difficult position: if they need to troubleshoot an issue, they can't access v1 documentation. 
```

Comment:
While this is apparently not transparent enough given you did not find it, the legacy docs which were homed at https://docs.fluxcd.io are still there, and we have preserved all existing hyperlinks (hopefully). We do not draw attention to them and in fact we've done everything we can to downplay their SEO, which also has the unfortunate side-effect that they are not searchable anymore outside of the site.

I was pretty sure we had rendered them completely un-searchable because of the difficulty of maintaining two parallel search engines for the same domain. But I just checked, and users who find https://docs.fluxcd.io or https://fluxcd.io/legacy/flux/ that it redirects to, can in fact search Flux v1 docs, and they appear to be every bit as functionally accessible as they ever were.

Hope this helps! Thanks!

Reply:
I had a review holding in mind a question, "how are users (that inherit a Flux v1 installation, lets say) supposed to find the legacy docs" and it looks like there's now only one straightforward way, on the front page, about 11 pages of content in (or about 80% of the way down the scroll bar.) I opened an issue so we can address this 👍 https://github.com/fluxcd/website/issues/689